fakenet/listeners/FTPListener.py

```diff
@@ -190,15 +206,33 @@ def chdir(self, path):
         if not self.lexists(path):
             path = '.'
 
+        # Collect NBIs
+        nbi = {"command": "CWD", "path": path}
+        collect_nbi(self.cmd_channel.remote_port, nbi,
+                self.cmd_channel.server.config.get('usessl'),
+                self.cmd_channel.server.diverterListenerCallbacks)
+
         return super(FakeFS, self).chdir(path)
 
     def remove(self, path):
 
+        # Collect NBIs
+        nbi = {"command": "DELETE", "filename": path}
+        collect_nbi(self.cmd_channel.remote_port, nbi,
+                self.cmd_channel.server.config.get('usessl'),
+                self.cmd_channel.server.diverterListenerCallbacks)
+
         # Don't remove anything
         pass
 
     def rmdir(self, path):
 
+        # Collect NBIs
+        nbi = {"command": "RMD", "directory": path}
```

Comment:
same as `DELETE` command, need to get the actual directory name and not the virtual path.